lib/registration.pm

```diff
@@ -100,29 +100,29 @@ sub fill_in_registration_data {
                     type_string $regcode;
                     sleep 1;
                     save_screenshot;
-                    send_key 'alt-n';     # next
                 }
             }
+            send_key 'alt-n';    # next
```

Comment:
Please see this:
http://147.2.207.208/tests/851/modules/register_via_scc_smt/steps/44

If we put the next in the loop in the case of multiple regcodes input, first regcode typed and then press next, it would contact scc server to register, when first product registration done, it would continue to register following products which were not input regcode. 
Like this example:
http://147.2.207.208/tests/822/modules/register_via_scc_smt/steps/50

So I put the next out of the loop for typing all regcodes and then register them in sequence.